internal/tenant/tenant.go

```diff
@@ -62,3 +63,38 @@ func (t tenantService) GetTenantInfoByNamespace(apiURL string, ns string) (InfoL
 
 	return tenantInfo, nil
 }
+
+// returns true if the cluster the ns is on has reached maximum capacity
+func (t tenantService) HasReachedMaxCapacity(apiURL, ns string) (bool, error) {
+
+	ti, err := t.GetTenantInfoByNamespace(apiURL, ns)
+	if err != nil {
+		return true, err
```

Comment:
returns true here means that the cluster has reached maximum capacity, along with an error, sort of does not make sense. Should this not be `false, err`?

Reply:
No, true, to err on the side of caution. I.E. if we can't find out about the tenant information, treat it as full so that we don't allocate unless sure. Moreover the caller is expected to handle errors :)

Reply:
Yes I think that returning `true` is safer in this case.

Reply:
ack